pkg/blockbuilder/types/grpc_transport.go

```diff
@@ -70,7 +70,12 @@ func (t *GRPCTransport) SendGetJobRequest(ctx context.Context, req *GetJobReques
 		BuilderId: req.BuilderID,
 	}
 
-	resp, err := t.GetJob(ctx, protoReq)
+	client, err := t.GetJob(ctx, protoReq)
```

Comment:
I don't think creating streaming clients for every RPC is any better than using the non-streaming RPCs we had prior. These are generally run in a loop